src/collapse/dtrace.rs

```diff
@@ -240,7 +240,7 @@ impl CollapsePrivate for Folder {
         self.opt.nthreads
     }
 
-    fn set_nthreads_(&mut self, n: usize) {
```

Comment:
Some of these changes seem unrelated to this commit?

Reply:
Well now that sample is single-threaded, we don't want a `set_nthreads` method on `Collapse` anymore, because you can be a `Collapse` and not have an option to set different numbers of threads.